server/server.go

```diff
@@ -398,6 +401,18 @@ func NewServer(configFile string) (*Server, error) {
 			SkipLogging: true,
 		}))
 	}
+	etcdKey := config.GetString("healthcheck/etcd_key", "/gohan")
```

Comment:
`NewServer` is more than 200 lines long. Could we move this config parsing inside `NewHealthCheck`?